tuneme/templates/core/article_page.html

```diff
@@ -23,7 +23,7 @@
   {% endif %}
 
   {% if self.image %}
-    {% image self.image width-98 as tmp_photo %}
+    {% image self.image width-620 as tmp_photo %}
```

Comment:
This is terrible for feature phone - you'll end up with a huge image downloaded for low end users